frontend/javascripts/oxalis/view/right-border-tabs/meshes_view.js

```diff
@@ -425,11 +456,12 @@ class MeshesView extends React.Component<Props, State> {
         meshfileResolutionIndex,
       );
 
-      if (this.props.segmentationLayer != null) {
+      if (this.props.visibleSegmentationLayer != null) {
         const job = await startComputeMeshFileJob(
           this.props.organization,
           this.props.datasetName,
-          this.props.segmentationLayer.fallbackLayer || this.props.segmentationLayer.name,
+          this.props.visibleSegmentationLayer.fallbackLayer ||
```

Comment:
This could reuse `getBaseLayerName(segmentationLayer: APIDataLayer)` from `meshes_view_helper.js`